config/base.yaml

```diff
@@ -10,6 +10,9 @@ logging:
   level: <LOG_LEVEL>
 security_server:
 - api_key: X-Road-apikey token=<API_KEY>
+  certificates:
```

Comment:
you have placed the "certificates" section only for the first security server, what about the other one?

Reply:
1. Removed irrelevant config from ``test_base_controller``
2. Added 
3. Removed 
4. These are enabled, couple of them indeed i had not noticed and fixed them. Note that I have ignored some of the more ones and these checks do not completely match the ones used during build process.
5. Cannot reproduce single test run failure, please give command-line failure example.